app/src/main/java/ch/epfl/qedit/view/edit/EditMapsActivity.java

```diff
@@ -66,24 +70,39 @@ public void onMapReady(GoogleMap googleMap) {
     public boolean onCreateOptionsMenu(Menu menu) {
         // TODO in future PR: Add a new item to go back to current location
         getMenuInflater().inflate(R.menu.edit_menu, menu);
+        MenuItem next = menu.findItem(R.id.next);
+        MenuItem previous = menu.findItem(R.id.previous);
+        MenuItem overview = menu.findItem(R.id.overview);
+
+        // We don't need below for the map
+        next.setVisible(false);
+        previous.setVisible(false);
+        overview.setVisible(false);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
+        int id = item.getItemId();
+
+        switch (id) {
+            case R.id.done:
+                onBackPressed();
+                break;
+            default:
+                break;
+        }
```

Comment:
I supposed that it is the default format for a onOptionsItemSelected method, but i think that a if is more appropriate